src/components/sections/SubSectionButtons.vue

```diff
@@ -0,0 +1,40 @@
+<script setup>
+const props = defineProps({
+  subsection: {
+    type: Object,
+    required: true
+  }
+})
+
+const goToPage = (page) => {
+  window.location.href = page
+}
+</script>
+
+<template>
+  <div class="datagouv-components fr-col-md-12">
```

Comment:
Peut-être possible d'utiliser un `<GenericContainer>` au niveau de la home page qui place `.datagouv-components` pour toute la page, et évite donc de le répéter dans tous les sous composants. Cf https://github.com/opendatateam/udata-front-kit/pull/473

Reply:
no need for a GenericContainer here I think, I just implemented generic classes in home component in a parent div